core-modules/package-lock.json

```diff
@@ -14,7 +14,7 @@
       "resolved": "https://registry.npmjs.org/@skpm/console/-/console-0.1.6.tgz",
       "integrity": "sha512-dquBerg7VngZM7LBjMqVjuVdOgmPugoLyGUH3onQuY9Lykw/x0/E5D7hazPH+lVWyyJ8zZfl7rNVSuYX336T5w==",
       "requires": {
-        "@skpm/util": "^0.1.9"
+        "@skpm/util": "0.1.12"
```

Comment:
I don't think it's that simple: my npm version (v6, latest) generated this file (which is correct: https://github.com/skpm/console/blob/master/package.json#L28) and it seems that a previous version of npm (v5) generates a slightly different file.
So that's not a fix for everybody. I'll trying to think of a more general solution